.gitignore

```diff
@@ -1,6 +1,8 @@
 *.swp
 data/
-project.xml
+standard.xml
 maxspeed.xml
 signals.xml
 *.mapcss
+Makefile
```

Comment:
Do we really want to ignore the Makefile? Shouldn't it be tracked by git?

Reply:
yes, this line must go away